packages/edit-site/src/components/sidebar-edit-mode/page-panels/style.scss

```diff
@@ -5,6 +5,37 @@
 	overflow: hidden;
 }
 
-.edit-site-page-panels__edit-template-button {
+.edit-site-page-panels__edit-template-button,
+.edit-site-page-panels__swap-template-button {
 	justify-content: center;
+	flex: auto;
+}
+
+
+.edit-site-page-panels__swap-template__modal-content .block-editor-block-patterns-list {
```

Comment:
If I'm not wrong, this CSS is repeated in a lot of places, I wonder if we should have this in the component itself (like gridColumns prop or something like that)

Reply:
That's true, but I focused for now with the technical implementation for the first round of reviews.